cip-constitution-tech/README.md

```diff
@@ -0,0 +1,255 @@
+
+When supplying a constitution hash digest to chain, the algorithm used MUST be Blake2b-256.
+Before creating a hash digest the constitution plain text must be in its raw un-rendered format.
+
+- add a way to change algorithm?
+
+### Storage
+
+The each ratified constitution MUST be stored, immutably on a distributed storage mechanism (such as IPFS).
+Where backups can be easily made in a permission less manner by interested parties.
+Must be easily accessible, normal tooling can support it.
+Authors SHOULD NOT specify the use of centralized gateways to access the constitution plain text.
+
+### Rich Text Formatting
+
+The constitution text MAY include a subset of markdown text styling.
+Tooling rendering the constitution for users, should recognize these and render them faithfully.
+
+#### Headers
+
+```md
+# H1
+## H2
+### H3
+#### H4
+##### H5
+###### H6
+
+Alternatively, for H1 and H2, an underline-ish style:
+
+Alt-H1
+======
+
+Alt-H2
+------
+```
+
+#### Emphasis
+
+```md
+Emphasis, aka italics, with *asterisks* or _underscores_.
+
+Strong emphasis, aka bold, with **asterisks** or __underscores__.
+
+Combined emphasis with **asterisks and _underscores_**.
+
+Strikethrough uses two tildes. ~~Scratch this.~~
+```
+
+#### Lists
+
+```md
+1. First ordered list item
+2. Another item
+⋅⋅* Unordered sub-list. 
+1. Actual numbers don't matter, just that it's a number
+⋅⋅1. Ordered sub-list
+4. And another item.
+```
+
+#### Links
+
+```md
+[I'm an inline-style link](https://www.google.com)
+```
+
+#### Code and Syntax Highlighting
+
+```md
+Inline `code` has `back-ticks around` it.
+```
+
+#### Tables
+
+```md
+Colons can be used to align columns.
+
+| Tables        | Are           | Cool  |
+| ------------- |:-------------:| -----:|
+| col 3 is      | right-aligned | $1600 |
+| col 2 is      | centered      |   $12 |
+| zebra stripes | are neat      |    $1 |
+
+There must be at least 3 dashes separating each header cell.
+The outer pipes (|) are optional, and you don't need to make the 
+raw Markdown line up prettily. You can also use inline Markdown.
+
+Markdown | Less | Pretty
+--- | --- | ---
+*Still* | `renders` | **nicely**
+1 | 2 | 3
+```
+
+#### Blockquotes
+
+```md
+> Blockquotes are very handy in email to emulate reply text.
+> This line is part of the same quote.
+
+Quote break.
+
+> This is a very long line that will still be quoted properly when it wraps. Oh boy let's keep writing to make sure this is long enough to actually wrap for everyone. Oh, you can *put* **Markdown** into a blockquote. 
+```
+
+#### Horizontal Rule
+
+```md
+Three or more...
+
+---
+
+Hyphens
+
+***
+
+Asterisks
+
+___
+
+Underscores
+```
+
+#### Line Breaks
+
+```md
+Here's a line for us to start with.
+
+This line is separated from the one above by two newlines, so it will be a *separate paragraph*.
+
+This line is also a separate paragraph, but...
+This line is only separated by a single newline, so it's a separate line in the *same paragraph*.
+```
+
```

Comment:
Are page breaks possible? If so, this would be better than using several line breaks.

Reply:
I dont believe there is, but if it was really wanted we could define a way to do page breaks